lsp-mode.el

```diff
@@ -6982,6 +6987,7 @@ argument ask the user to select which language server to start. "
   (when (and lsp-auto-configure)
     (seq-do (lambda (package) (require package nil t))
             lsp-client-packages))
+  (run-hooks 'lsp-after-client-packages-hook)
```

Comment:
that would run every time you run M-x lsp. So we should put in the `when` and also make sure that the require block is called only once.